api/src/db/seeder.js

```diff
@@ -0,0 +1,73 @@
+const mongoose = require('mongoose')
+const Account = require('./Account')
+const Wish = require('./Wish')
+const chance = require('chance').Chance()
+const wishRouter = require('../routes/wish')
+var _ = require('lodash');
+
+
+process.env.mongoUrl = process.env.mongoUrl || 'mongodb://localhost:27017/my_database'
+require('./bootstrap-mongoose')
+
+createAccounts = async () => {
+
+  await Account.find({}).remove(async function() {
+
+    const userAdmin = new Account({username: 'Admin', role: 'admin'});
+    await userAdmin.setPassword('Password321');
+    await userAdmin.save();
+    await Account.authenticate()('Admin', 'Password321');
+
+    let i = 0
+    let count = 10
+    while (i < count) {
+      let username = chance.email();
+      const user = new Account({
+        provider: 'local',
+        username: username,
+        firstName: chance.first(),
+        lastName: chance.last(),
+        password: 'Password123'
+      });
+      await user.setPassword('Password123');
+      await user.save();
+      await Account.authenticate()(username, 'Password123');
```

Comment:
Do we need to Authenticate here?

Reply:
We don't need it. I will remove line 34.